controllers/automation/packagedeployment_controller.go

```diff
@@ -207,6 +210,9 @@ func (r *PackageDeploymentReconciler) applyPackageMutations(ctx context.Context,
 					Namespace: n.GetNamespace(),
 				},
 			}
+			if id.Namespace == "" {
```

Comment:
Can't review unchanged lines, but a few lines up I still see the old annotation being used - I think you need to rebase on the latest main, I merged a change to the annotation.

Also, in that change I look explicitly for "True" - if you have a chance maybe you can fix that to parse it to a bool and check it, rather than the string "True".

Reply:
Rebased and changed it to parse the string to a bool and check the bool value.